app/assets/stylesheets/govuk-component/_govspeak.scss

```diff
@@ -8,6 +8,12 @@
     text-align: start;
   }
 
+  &.rich-govspeak {
+    strong {
+      font-weight: bold;
```

Comment:
Presumably somewhere there's a reset that removes the default `strong` browser rendering?